app/views/users/_user.html.erb

```diff
@@ -98,16 +100,18 @@
     <div class="row mb-4">
       <div class="col">
         <% follow_request = current_user.sent_follow_requests.find_by(recipient: @user) %>
-        <% if follow_request %>
-          <%= link_to follow_request, data: { turbo_method: :delete }, class: "btn btn-outline-secondary btn-block" do %>
-            <% if follow_request.pending? %>
-              Un-request
-            <% else %>
-              Un-follow
+        <% if @user != current_user %>
```

Comment:
nice way to hide pending requests for users who are looking at the profile that's not their own!